lib/improver/tests/nowcasting/optical_flow/test_OpticalFlow.py

```diff
@@ -654,19 +654,19 @@ def test_values(self):
         self.assertAlmostEqual(np.mean(vcube.data), 2.1719084)
 
     def test_update_smoothing_radius(self):
-        """Test data smoothing radius is updated if cube time difference is not
-        15 minutes.  We don't care about the error this trips, we just want to
-        make sure the radius is updated correctly."""
+        """Test data smoothing radius is updated if cube time difference is
+        greater than 15 minutes.  The updated radius value (12 km = 6 grid
+        squares) causes an error to be raised."""
         time_unit = self.cube2.coord("time").units
         new_time = time_unit.num2date(self.cube2.coord("time").points[0])
         new_time += datetime.timedelta(seconds=900)
         self.cube2.remove_coord("time")
         time_coord = DimCoord(time_unit.date2num(new_time),
                               standard_name="time", units=time_unit)
         self.cube2.add_aux_coord(time_coord)
-        with self.assertRaises(ValueError):
+        msg = "data smoothing radius 6"
```

Comment:
I think we should change this msg to specify that it is being raised because the box size is too small, not because the adjusted radius is too small now.